### ISSUE TITLE
Error in auto-generated code

### ISSUE THREAD

**Issue by xauxatz** [perak/kitchen-site#154]
After 0.9.52 I am getting a strange error.
In the top of styles.less we have:
@import "{}/client/styles/framework/bootstrap3/custom.bootstrap.less";
This gives me an error with WebStorm that it can't find custom.bootstrap.less. I've checked and it's there.
If I remove the {} it works fine.

**Comment by perak** [comment]
@xauxatz This error appears only when application is started for the very first time. Few seconds later, `nemo64:bootstrap` automatically creates file and meteor restarts with no errors. 

Since Meteor 1.2 string `{}` in `@import` path tells less compiler that path is relative to application root.

Are you using Meteor 1.2 ?

**Comment by xauxatz** [comment]
Yes, I'm using Meteor 1.2. And it remains a problem until I remove the {}.
Perhaps it's because I'm running the app from WebStorm.

**Comment by perak** [comment]
humm... I am just playing with that, I'l let you know

**Comment by perak** [comment]
You're right - `{}` is unnecessary, I just removed it, will be available in next version `v0.9.53`. 

Until then, you can manually remove `{}` from ~/.meteor-kitchen/templates/ui/bootstrap3/styles.less` file.

**Comment by perak** [comment]
(when meteor 1.2.0 is released, it didn't work without {} - who knows why...)

**Comment by xauxatz** [comment]
Ok great :-)
